src/handlers.js

```diff
@@ -166,14 +192,26 @@ async function statusHandler(message) {
 async function jobHandler(message) {
   let context = this.context;
 
+  // Authenticating as installation.
+  try {
```

Comment:
Same here -- remove the try/catch and debug logging, letting `getInstallationGithub` do the logging.